src/Lucene.Net/Index/CompoundFileReader.cs

```diff
@@ -166,7 +166,17 @@ public override IndexInput OpenInput(System.String id, int readBufferSize, IStat
 				return new CSIndexInput(stream, entry.offset, entry.length, readBufferSize, state);
 			}
 		}
-		
+
```

Comment:
Note that we should have *no* use of compound files in ravendb

Reply:
We use a compound file for merged segments (on by default):
https://github.com/ravendb/ravendb/blob/05e95d4bf2ca6c9db8515190f57ca113511b48f0/src/Raven.Server/Config/Categories/IndexingConfiguration.cs#L444